clients/broadcaster.go

```diff
@@ -64,6 +70,81 @@ type EncodedProfile struct {
 	ChromaFormat int64  `json:"chromaFormat,omitempty"`
 }
 
+// DefaultTranscodeProfiles defines the default set of encoding profiles to use when none are specified
+var DefaultTranscodeProfiles = []EncodedProfile{
+	// {
+	// 	Name:    "240p0",
+	// 	FPS:     0,
+	// 	Bitrate: 250_000,
+	// 	Width:   426,
+	// 	Height:  240,
+	// },
+	{
+		Name:    "360p0",
+		FPS:     0,
+		Bitrate: 800_000,
+		Width:   640,
+		Height:  360,
+	},
+	// {
+	// 	Name:    "480p0",
+	// 	FPS:     0,
+	// 	Bitrate: 1_600_000,
+	// 	Width:   854,
+	// 	Height:  480,
+	// },
+	{
+		Name:    "720p0",
+		FPS:     0,
+		Bitrate: 3_000_000,
+		Width:   1280,
+		Height:  720,
+	},
+}
+
+func GetPlaybackProfiles(iv InputVideo) ([]EncodedProfile, error) {
```

Comment:
I moved these functions to avoid an import cycle, perhaps I should move them to a whole new package though?